src/Defaults/baileys-version.ts

```diff
@@ -0,0 +1 @@
+export const version = [2, 3000, 1023223821] as const;
```

Comment:
```suggestion
export const version = [2, 3000, 1023223821] as const;
```

We need to update the github action to support this